src/exporters/MMDExporter.ts

```diff
@@ -1,18 +1,23 @@
-import { Matrix4, Quaternion, Vector3 } from 'three'
+import { Matrix4, Quaternion, Vector3, Bone, SkinnedMesh } from 'three'
+// @ts-ignore
```

Comment:
Since the type definition for `mmd-parser` does not exist, I added `@ts-ignore` and defined the `interface` in a later [section](https://github.com/pmndrs/three-stdlib/pull/74/files#diff-335dadb8dd8be15fe700cd022ffa48ab7a16a306d0f621fa8b3151381da7b39eR10).

Reply:
You can add a `custom.d.ts` to the root of the project and declare the module there, I would prefer this over a `ts-ignore`